expyfun/stimuli/_hrtf.py

```diff
@@ -151,3 +151,67 @@ def convolve_hrtf(data, fs, angle, source='barb'):
         brir = [resample(b, fs, brir_fs) for b in brir]
     out = np.array([np.convolve(data, brir[o]) for o in order])
     return out
+
+    
+def make_sym(x):
+    """
+    forces symmetry around zero
+    """
+    n = x.shape[-1]
+    x[..., :-((n + 1) // 2):-1] = np.conj(x[..., 1:(n + 1) // 2])
+    if np.mod(n, 2) == 0:
+        x[..., n // 2] = 0
+    return x
+    
+
+def interp_hrtf(angle, fs):
```

Comment:
Probably in `_get_hrtf`. It also has the list of angles, so you shouldn't need to call `convolve_hrtf` from within your function (this would be bad software design anyway)